build.sh

```diff
@@ -343,8 +346,9 @@ echo ""
 if [[ $JF_SKIP_DOCS != [yY]* ]]; then
     if hash ford 2>/dev/null; then
     echo "Building documentation..."
-    [[ $TRY_UNICODE = [yY]* ]] && MACRO_FLAG="-m USE_UCS4"
-    ford $MACRO_FLAG -p $PAGESDIR $FORDMD
+    [[ $TRY_UNICODE = [yY]* ]] && MACRO_FLAG=("-m" "USE_UCS4")
+    echo "$FPP" > preprocessor-def.md # Override via include in project file, until FORD gets CLI for this
+    ford --debug "${MACRO_FLAG[@]}" -p "$PAGESDIR" "$FORDMD"
```

Comment:
We may want to remove the `--debug` flag here... it generates a bunch of output... however, I want to leave it in the CMake build script to ease Homebrew debugging...